ex1/functions.py

```diff
@@ -46,15 +46,97 @@ def f():
 ##############################
 
 # Define a function named add that takes two numbers and returns the sum.
+def add(a,b):
+	"""Returns the sum of two given numbers.
+	
+	Adds variables a and b an returns the result:
+	
+	Parameters
+	----------
+	a : int	
+		First number to be added to.
+	b : int	
+		Number added to a.
+	
+	Returns
+	-------
+	sum : int
+		Result of addition
+	"""
+	
+	sum = a + b
+	return sum
 
 # Define a function named to_tuple that takes three arguments and returns a
 # tuple of these three arguments.
+def to_tuple(arg1, arg2, arg3):
+	"""Returns a tuple of arguments
+	
+	Takes 3 arguments and returns a tupel.
+	
+	Parameters
+	----------
+	arg1, arg2, arg3 : optional
+		Arguments formed into a tupel.
+		
+	Returns
+	-------
+	tuple : tuple
+		Tuple of all the arguments
+	"""
+	tuple = (arg1, arg2, arg3)
+	return tuple
 
 # Define a function named check5 that checks if a number is greater than 5 and
 # returns True or False.
+def check5(a):
+	"""Checks if number is greater than a reference number.
+	
+	Takes a given number to check if it is greater than 5.
+	
+	Parameters
+	----------
+	a : int
+		Number given to be checked.
+		
+	Returns
+	-------
+	True	
+		If number is greater than 5.
+	False	
+		If number is smaller than 5.
+	
+	"""
+	if (a>5):
```

Comment:
no need to have a separate if else clause for such simple cases. See #20 for just returning the comparison result.